build/yamls/antrea-windows-containerd-with-ovs.yml

```diff
@@ -22,6 +22,145 @@ data:
+            Log "SHA256 mismatch for OVS download"
+            exit 1
+        }
+    }
+
+    WaitExpandFiles $OVSZip $OVSDownloadDir
+    # Copy OVS package to target dir.
+    Log "Copying OVS package from $OVSDownloadDir\openvswitch to $OVSInstallDir"
+    mv "$OVSDownloadDir\openvswitch" $OVSInstallDir
+    rm $OVSZip
+    # Create log and run dir.
+    $OVS_LOG_PATH = $OVSInstallDir + "\var\log\openvswitch"
+    CreatePath $OVS_LOG_PATH
+    $OVSRunDir = $OVSInstallDir + "\var\run\openvswitch"
+    CreatePath $OVSRunDir
+    $OVSDriverDir = "$OVSInstallDir\driver"
+
+    # Install OVS driver certificate.
+    $DriverFile="$OVSDriverDir\OVSExt.sys"
+    if ($ImportCertificate) {
+        $CertificateFile = "$OVSDriverDir\package.cer"
+        if (!(Test-Path $CertificateFile)) {
+            $ExportType = [System.Security.Cryptography.X509Certificates.X509ContentType]::Cert;
+            $Cert = (Get-AuthenticodeSignature $DriverFile).SignerCertificate;
+            [System.IO.File]::WriteAllBytes($CertificateFile, $Cert.Export($ExportType));
+        }
+        Log "Installing OVS driver certificate."
+        Import-Certificate -FilePath "$CertificateFile" -CertStoreLocation cert:\LocalMachine\TrustedPublisher
+        Import-Certificate -FilePath "$CertificateFile" -CertStoreLocation cert:\LocalMachine\Root
+    }
+
+    # Install Microsoft Visual C++ Redistributable Package.
+    if (Test-Path $OVSInstallDir\redist) {
+        Log "Installing Microsoft Visual C++ Redistributable Package."
+        $RedistFiles = Get-ChildItem "$OVSInstallDir\redist" -Filter *.exe
+        $RedistFiles | ForEach-Object {
+            Log "Installing $_"
+            Start-Process -FilePath $_.FullName -Args '/install /passive /norestart' -Verb RunAs -Wait
+        }
+    }
+
+    # Install powershell modules
+    if (Test-Path $OVSInstallDir\scripts) {
+        Log "Installing powershell modules."
+        $PSModuleFiles = Get-ChildItem "$OVSInstallDir\scripts" -Filter *.psm1
+        $PSModuleFiles | ForEach-Object {
+            $PSModulePath = Join-Path -Path $PowerShellModuleBase -ChildPath $_.BaseName
+            if (!(Test-Path $PSModulePath)) {
+                Log "Installing $_"
+                mkdir -p $PSModulePath
+                Copy-Item $_.FullName $PSModulePath
+            }
+        }
+    }
+
+    # Install OVS kernel driver.
+    Log "Installing OVS kernel driver"
+    $VMMSStatus = $(Get-Service vmms -ErrorAction SilentlyContinue).Status
+    if (!$VMMSStatus) {
+        $VMMSStatus = "not exist"
+    }
+    Log "Hyper-V Virtual Machine Management service status: $VMMSStatus"
+    if ($VMMSStatus -eq "Running") {
+        cmd /c "cd $OVSDriverDir && install.cmd"
+    } else {
+        cd $OVSDriverDir ; netcfg -l .\ovsext.inf -c s -i OVSExt; cd $WorkDir
+    }
+    if (!$?) {
+        Log "Install OVS kernel driver failed, exit"
+        exit 1
+    }
+    $OVS_BIN_PATH="$OVSInstallDir\usr\bin;$OVSInstallDir\usr\sbin"
```

Comment:
bin/sbin path setting is not needed ? mountpath for bin/sbin is already set at later point

Reply:
that is for adding $mountpath/openvswitch/usr/bin to path ... This is for adding C:/openvswitch/usr/bin to path

Reply:
Is it required to add `C:/openvswitch/usr/bin` and `C:/openvswitch/usr/sbin` to PATH? 
I think we don't need to run OVS commands on host, so it shouldn't be required.